src/Micrometa/Infrastructure/Parser/JsonLD/CachingContextLoader.php

```diff
@@ -73,6 +73,11 @@ public function __construct(VocabularyCache $vocabularyCache)
      */
     public function loadDocument($url)
     {
+        // ML\JsonLD\FileGetContentsLoader is no longer able to fetch schema.org, this is a workaround
+        if (in_array(strtolower(rtrim($url, '/')), ['http://schema.org', 'https://schema.org'])) {
```

Comment:
Based on the issue description the fix hopefully is temporary. 

Would it be possible to use `get_headers` to check whether the `Link` header is there first?

Reply:
Yes, that's the ideal situation. `ML\JsonLD` doesn't seem to be very active, so I am not sure if the will eventually make a fix.  This was just a quick workaround because the current way of doing it will no longer work.

The easiest way would be to use Guzzle and do a `HEAD` request to check for links, which Guzzle parser.  Not sure if you want to add that dependency or write your own parser.

Reply:
As far my experience with `get_headers` goes using that built-in method would be sufficient.

We have a hard dependency on `ext-curl` via `league/uri`, so for the sake of this being a hotfix I'd then be in favor of adding a `curl` implementation.